packages/fx-core/resource/package.nls.json

```diff
@@ -51,15 +51,12 @@
   "core.consolidateLocalRemote.SuccessMessage": "Congratulations! You have successfully upgraded your Teams App project structure. We have added config.local.json for local debug and manifest.template.json for both Teams app running locally and remotely. And we have backed up localSettings.json, manifest.local.template.json and manifest.remote.template.json to .backup folder.",
-  "core.aadManifestMigration.Canceled": "[core] Upgrade cancelled.",
-  "core.aadManifestMigration.VSCodeCanceledNotice": "[core] Notice upgrade to new configuration files is a must-have to continue to use current version Teams Toolkit. If you are not ready to upgrade and want to continue to use the old version Teams Toolkit, please find Teams Toolkit in Extension and install the version <= ????",
-  "core.aadManifestMigration.CLICanceledNotice": "[core] Notice upgrade to new configuration files is a must-have to continue to use current version Teams Toolkit CLI. If you want to upgrade, please trigger this command again.",
-  "core.aadManifestMigration.CLINotReadyInstallLatestVersionNotice": "[core] If you are not ready to upgrade and want to continue to use the old version Teams Toolkit CLI, please install the version <= ????",
-  "core.aadManifestMigration.Message": "Teams Toolkit will upgrade your project's to support the latest features. The upgrade process will not change your custom code and create the backup files in case revert is needed.\nNotice this upgrade is a must-have to continue to use current version Teams Toolkit. If you are not ready to upgrade and want to continue to use the old version, please find Teams Toolkit in Extension and install the version <= ????",
-  "core.aadManifestMigration.outputMsg": "Congratulations! You have successfully upgraded your Teams App project structure. We have added aad.template.json for AAD app configuration. And we have backed up projectSettings.json, permissions.json to .backup folder.",
-  "core.aadManifestMigration.SuccessMessage": "Upgrade succeeded. Your project is upgraded to new configuration files.",
+  "core.consolidateLocalRemoteWithAadManifest.SuccessMessage": "Congratulations! You have successfully upgraded your Teams App project structure. We have added config.local.json for local debug, manifest.template.json for both Teams app running locally and remotely and aad.template.json for AAD manifest template. And we have backed up localSettings.json, manifest.local.template.json, manifest.remote.template.json, projectSettings.json to .backup folder.",
+  "core.aadManifestMigration.outputMsg": "Congratulations! You have successfully upgraded your Teams App to support AAD manifest feature. We have added aad.template.json and updated your projectSettings.json file. And we have backed up projectSettings.json to .backup folder.",
```

Comment:
Since we don't want to treat only aad manifest as "upgrade". Can we avoid this wording?

Reply:
Thanks updated